config/migrations/2024/20240412000000-mandataris-publication-status.sparql

```diff
@@ -21,7 +23,8 @@ INSERT DATA {
         skos:Concept,
         extlmb:MandatarisPublicationStatusCode;
       skos:prefLabel "Draft";
-      mu:uuid "588ce330-4abb-4448-9776-a17d9305df07".
+      mu:uuid "588ce330-4abb-4448-9776-a17d9305df07";
+      sh:order "1"^^xsd:integer.
```

Comment:
Don''t we need a new migration to add this? 😅  For example the migration is not run on my machine because it already ran previously

Reply:
Sorry but we need a new migration 🙈

Reply:
Fixed